Client.xcodeproj/project.pbxproj

```diff
@@ -3740,7 +3736,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "/usr/local/bin/carthage copy-frameworks";
+			shellScript = "/usr/local/bin/carthage copy-frameworks\n";
```

Comment:
Know anything about this?

Reply:
🤔 Nope, no idea why it added this. I can remove it.